apps/hip/templates/includes/header.html

```diff
@@ -1,33 +1,47 @@
 {% load static %}
 
-<nav class="navbar navbar-hip" role="navigation" aria-label="main navigation">
-  <div class="navbar-brand">
-    <div class="navbar-item">
-      <a class="button logo-hip" href="/home">
-        <strong>Logo</strong>
+<nav class="navbar columns is-desktop navbar-hip" role="navigation" aria-label="main navigation">
+    <div class="is-hidden-desktop is-full-width-hip is-full-height-hip columns push-to-back-hip is-mobile">
+      <div class="column is-2 is-jusify-content-flex-start">
+        <a role="button" id="navbar-burger" class="navbar-burger is-block is-size-5 mr-2 mt-4 is-mobile" aria-label="menu" aria-expanded="false" data-target="sidebarContent">
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+      <div class="column is-8">
+        <a class="logo-hip" href="https://www.phila.gov/">
```

Comment:
I know this is a post-sprint change, but let's make the logo go to `/home` since the client approved that change and it will make it easier for QA to move around the site by clicking the logo, rather than having to use the URL bar. (Can be done in a future PR)

Reply:
done